### ISSUE TITLE
Watch test set error during training

### ISSUE THREAD

**Issue by RayFireh** [Microsoft/CNTK#567]
Is it possible to run test after each epoch to see how the error on a testing set changes? Because currently it shows training set error only.
And if it possible any example configuration?

**Comment by frankseide** [comment]
Yes. You will just need to define a second `reader` section in your CNTK config file, under the name `cvReader`, which points to your development test set. For example, see the [SequenceToSequence example](https://github.com/Microsoft/CNTK/blob/master/Examples/SequenceToSequence/CMUDict/Config/G2P.cntk).

**Comment by RayFireh** [comment]
Thank You for your prompt response. Looks like I have missed it somehow. Will try it tomorrow.
